src/Components/EventCreation/Form/Form.js

```diff
@@ -9,30 +9,39 @@ import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
 import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
 import { TimePicker } from "@mui/x-date-pickers/TimePicker";
 import { DatePicker } from "@mui/x-date-pickers/DatePicker";
+import { createEvent } from "../../../queries";
 
-const Form = ({logoutUser}) => {
+const Form = ({logoutUser, loggedInUser}) => {
   const [game, setGame] = useState('');
   const [category, setCategory] = useState('');
-  const [address, setAddress] = useState('');
-  const [city, setCity] = useState('');
-  const [state, setState] = useState('');
-  const [zip, setZip] = useState('');
-  const [date, setDate] = useState('');
-  const [startTime, setStartTime] = useState('');
-  const [endTime, setEndTime] = useState('');
-  const [eventDescription, setEventDescription] = useState('');
+  const [address, setAddress] = useState();
+  const [city, setCity] = useState();
+  const [state, setState] = useState();
+  const [zip, setZip] = useState();
+  const [date, setDate] = useState();
+  const [startTime, setStartTime] = useState();
+  const [endTime, setEndTime] = useState();
+  const [eventDescription, setEventDescription] = useState();
 
   const handleSubmit = (event) => {
     event.preventDefault();
 
-    // const formData = {
-
-    // };
-
-    // useEffect(() => {
-
-    // })
+    createNewEvent({
+      variables: {
+        input: {
+          date: date,
+          address: address,
+          city: city,
+          state: state,
+          zip: zip,
+          title: game,
+          description: eventDescription,
+          hostId: loggedInUser,
+        },
+      },
+    });
   };
+  const [createNewEvent, { data: data, loading: loading, error: error }] = useMutation(createEvent);
 
   return (
     <form className="form" onSubmit={handleSubmit}>
```

Comment:
Just want to say that this form looks amazing. Really nice job on the styling.

Reply:
I second this, great work on the styling!